worker/task_executor.go

```diff
@@ -0,0 +1,164 @@
+package worker
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
```

Comment:
overall, I'm missing a more general doc explaining the design of this file (e.g we copy script to a path, read the exit codes sent to a file ...)

Reply:
I've created a doc in its beginning, could you check please?